EIPS/eip-x.md

```diff
@@ -0,0 +1,140 @@
+| Blocks regularly under-full, user willing to wait for a good price. | User bids below the low end of the recently accepted prices, may need to wait for a while. If waiting too long, user may need to re-submit with a higher price. | User chooses their price, and waits for it, or manually re-submits.                          | User chooses their lowest price, but also their highest price and maximum wait time, so no resubmission is needed. |
+
+In all cases, the escalator algorithm as I have described is able to perform optimally.
+
+The current gas auction model works well under half-full and less conditions, but for users with urgent needs, has the downside of overpayment. For users seeking a low price, the current model has the downside of requiring re-submission, but has the benefit of always giving users a path towards reliable block inclusion.
+
+EIP-1559 also performs well under normal conditions, but under conditions where blocks are regularly full, the price discovery mechanism breaks, and miners will fall back to the `TIP` parameter to choose the transactions to include, meaning that under network congestion, EIP-1559 forces users to _either_ choose efficient prices or certainty of next-block inclusion.
+
+EIP-1559 also has all the re-submission issues of the current model in situations where a user would like to pay under the current market rate, but has certain time constraints limiting their patience. The Escalator algorithm is the only strategy listed here that allows users to discover the lowest possible price given the network conditions and their time constraints.
+
+## Specification
+<!--The technical specification should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations for any of the current Ethereum platforms (go-ethereum, parity, cpp-ethereum, ethereumj, ethereumjs, and [others](https://github.com/ethereum/wiki/wiki/Clients)).-->
+
+**Client-Wide Parameters**
+* `INITIAL_FORK_BLKNUM`: TBD
+
+**Transaction Parameters**
+The transaction `gasPrice` parameter is now optional, and if excluded can be replaced by these parameters instead:
+
+* `START_PRICE`: The lowest price that the user would like to pay for the transaction.
+* `START_BLOCK`: The first block that this transaction is valid at.
+* `MAX_PRICE`: The maximum price the sender would be willing to pay to have this transaction processed.
+* `MAX_BLOCK`: The last block that the user is willing to wait for the transaction to be processed in.
```

Comment:
I'm not a fan of the transaction getting dropped when max-block is reached.  In most cases I would want to escalate up to target price and then sit at that target price until mined or cancelled.  I can *also* imagine situations where I want to "mine by this block or give up", so I can appreciate the desire but I don't think it is the common case.

Reply:
Thanks for highlighting this distinction. I think I agree with you as the default case, especially since it resembles current behavior more, but I think there is a use case (for a future EIP, perhaps a new tx type) to allow TXs that become invalid over time or after some conditions. Making this change.